amundsen-kube-helm/templates/helm/neo4j/templates/deployment.yaml

```diff
@@ -24,6 +24,26 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
       {{- end }}
+      initContainers:
+        - name: init-neo4j-plugins
+          image: "appropriate/curl:latest"
+          imagePullPolicy: "IfNotPresent"
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              curl -L https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/3.3.0.4/apoc-3.3.0.4-all.jar -O
+              curl -L https://github.com/neo4j-contrib/neo4j-graph-algorithms/releases/download/3.3.5.0/graph-algorithms-algo-3.3.5.0.jar -O
+              curl -L http://central.maven.org/maven2/com/amazonaws/aws-java-sdk-core/1.11.250/aws-java-sdk-core-1.11.250.jar -O
```

Comment:
I think these maven URLs need https (and a different subdomain as indicated below); they return 501 rather than 3xx.

```
$ curl -L http://central.maven.org/maven2/com/amazonaws/aws-java-sdk-core/1.11.250/aws-java-sdk-core-1.11.250.jar
501 HTTPS Required.
Use https://repo1.maven.org/maven2/
More information at https://links.sonatype.com/central/501-https-required
```

Reply:
hey @bryfox thanks for catching that; although, I see that it works when you attach the `-O` flag; after viewing the man page for curl, I'm not sure why. Anyhow, I'll update it to be cautious.